src/bp/nlu-core/intents/exact-intent-classifier.ts

```diff
@@ -77,25 +83,33 @@ export class ExactIntenClassifier implements IntentClassifier {
       const model: Model = await validate(raw, modelSchema)
       this.model = model
     } catch (err) {
-      throw new ModelLoadingError(ExactIntenClassifier._name, err)
+      throw new ModelLoadingError(ExactIntenClassifier._displayName, err)
     }
   }
 
-  async predict(utterance: Utterance): Promise<IntentPredictions> {
+  async predict(utterance: Utterance): Promise<NoneableIntentPredictions> {
     if (!this.model) {
-      throw new Error(`${ExactIntenClassifier._name} must be trained before calling predict.`)
+      throw new Error(`${ExactIntenClassifier._displayName} must be trained before calling predict.`)
     }
 
+    const { _name: extractor } = ExactIntenClassifier
```

Comment:
why using a var for this ? is it because you're lazy using the auto complete twice down below XD ?

Reply:
no preferences, will change if you dont like it